DatadogSessionReplay/Sources/SessionReplay+objc.swift

```diff
@@ -146,6 +146,29 @@ public final class objc_SessionReplayConfiguration: NSObject {
         super.init()
     }
 
+    /// Creates Session Replay configuration.
+    ///
+    /// - Parameters:
+    ///   - replaySampleRate: The sampling rate for Session Replay. It is applied in addition to the RUM session sample rate.
+    ///   - textAndInputPrivacyLevel: The way texts and inputs (e.g. label, textfield, checkbox) should be masked. Default: `.maskAll`.
+    ///   - imagePrivacyLevel: Image recording privacy level. Default: `.maskAll`.
+    ///   - touchPrivacyLevel: The way user touches (e.g. tap) should be masked. Default: `.hide`.
```

Comment:
nit: I realize we mention default values for this API and the one above, but we don't provide any for now. Maybe we should remove them?

Reply:
Yes, we should remove them from the comment. Defaults values does not work in objc anyway..